pkgs/by-name/po/polybar-pulseaudio-control/package.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, bash
+, coreutils
+, fetchFromGitHub
+, gawk
+, makeWrapper
+, pulseaudio
+, stdenv
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "polybar-pulseaudio-control";
+  version = "3.1.1";
+
+  src = fetchFromGitHub {
+    owner = "marioortizmanero";
+    repo = "polybar-pulseaudio-control";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-egCBCnhnmHHKFeDkpaF9Upv/oZ0K3XGyutnp4slq9Vc=";
+  };
+
+  dontConfigure = true;
+  dontBuild = true;
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mv pulseaudio-control.bash pulseaudio-control
+    install -Dm755 pulseaudio-control -t $out/bin
```

This can proably be further reduced

```suggestion
    install -Dm755 pulseaudio-control.bash $out/bin/pulseaudio-control
```